spec/rails_helper.rb

```diff
@@ -9,6 +9,8 @@
 ENV['KEYCLOAK_ISS'] ||= 'http://localhost:8443/realms/coursemology_test'
 ENV['KEYCLOAK_AUD'] ||= 'account'
 ENV['KEYCLOAK_REALM'] ||= 'coursemology_test'
+# All the codaveri endpoints are mocked, so this URL should never be called
```

Comment:
If it's not called, why do we still need to specify it here?

Reply:
Apart from this, rest LGTM, update to a dummy value then we can merge it in.

Reply:
Done.

Reply:
can you squash your last 2 commits?
Also the env is still pointing to the live url instead of a dummy one.

Reply:
Done.

Had to make one more change: coursemology docker container test requires the real docker apis (or ping real docker service locally), so I had it set Excon to not mock requests